.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```

Comment:
This should fix the EOL issue that sometimes occurs when the code is written in diff computers.